internal/controlplane/handlers_ruletype.go

```diff
@@ -234,6 +234,13 @@ func (s *Server) DeleteRuleType(
 		return nil, status.Errorf(codes.Unknown, "failed to get rule type: %s", err)
 	}
 
+	// TEMPORARY HACK: Since we do not need to support the deletion of bundle
+	// rule types yet, reject them in the API
+	// TODO: Move this deletion logic to RuleTypeService
+	if rtdb.SubscriptionID.Valid {
+		return nil, status.Errorf(codes.InvalidArgument, "cannot delete rule type from bundle")
+	}
```

Comment:
Should this logic be added to `UpdateRuleType` as well?

Reply:
See the comment on the delete profile method